packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1127,6 +1155,9 @@ class SelectionOverlay {
     if (!listEquals(_selectionEndpoints, value)) {
       _markNeedsBuild();
     }
+    if (_isDraggingEndHandle || _isDraggingStartHandle) {
+      HapticFeedback.selectionClick();
```

Comment:
Is there any platform check that should happen here?  Or if not I guess that 1. iOS and Android both vibrate in the same case here and 2. calling this on platforms that don't vibrate, like desktop, is just a noop.

Reply:
Update: I tried this on native iOS and there is no vibration for changing the text selection at all, as far as I can tell.  So this should only happen on Android?